test/games/strategy/triplea/delegate/StratBombTest.java

```diff
@@ -0,0 +1,127 @@
+  }
+
+  private ITestDelegateBridge getDelegateBridge(final PlayerID player) {
+    return GameDataTestUtil.getDelegateBridge(player, m_data);
+  }
+
+
+  @Test
+  public void TestBombingRaid2targets() {
+    final Territory germany = m_data.getMap().getTerritory("Western Germany");
+    final Territory uk = m_data.getMap().getTerritory("United Kingdom");
+    final PlayerID germans = GameDataTestUtil.germans(m_data);
+    final PlayerID british = GameDataTestUtil.british(m_data);
+    // add a unit
+    final Unit stratBomber = GameDataTestUtil.bomber(m_data).create(british);
+    final Unit tacBomber1 = GameDataTestUtil.tacBomber(m_data).create(british);
+    final Unit tacBomber2 = GameDataTestUtil.tacBomber(m_data).create(british);
+    m_data.performChange(ChangeFactory.addUnits(uk, Collections.singleton(stratBomber)));
+    m_data.performChange(ChangeFactory.addUnits(uk, Collections.singleton(tacBomber1)));
+    m_data.performChange(ChangeFactory.addUnits(uk, Collections.singleton(tacBomber2)));
+    final BattleTracker tracker = new BattleTracker();
+    List<Unit> attackers = new ArrayList<>();
+    attackers.add(stratBomber);
+    attackers.add(tacBomber1);
+    attackers.add(tacBomber2);
+    HashMap<Unit, HashSet<Unit>> targets = new HashMap<>();
+    TripleAUnit airfield = null;
+    TripleAUnit harbour = null;
+    TripleAUnit factory = null;
+
+    // Find facilities in territory
+    for( final Unit target : germany.getUnits().getUnits() ) {
+      switch( target.getType().getName() ) {
+        case "airfield"      : airfield = (TripleAUnit) target; 
+          break;
+        case "harbour"       : harbour = (TripleAUnit) target;
+          break;
+        case "factory_major" : factory = (TripleAUnit) target;
+          break;
+      }
+    }
+    targets.put(airfield, new HashSet<>(Collections.singleton(tacBomber1)));
+    targets.put(harbour, new HashSet<>(Collections.singleton(tacBomber2)));
+    targets.put(factory, new HashSet<>(Collections.singleton(stratBomber)));
+    final ITestDelegateBridge bridge = getDelegateBridge(british);
+    tracker.addBattle(new RouteScripted(germany), attackers, true, british, bridge, null, null, targets, true);
+                
+    final StrategicBombingRaidBattle battle = new StrategicBombingRaidBattle(germany, m_data, british, tracker);
+    battle.addAttackChange(m_data.getMap().getRoute(uk, germany), uk.getUnits().getMatches(Matches.UnitIsStrategicBomber), null);
+    addTo(germany, uk.getUnits().getMatches(Matches.UnitIsStrategicBomber));
+    tracker.getBattleRecords().addBattle(british, battle.getBattleID(), germany, battle.getBattleType());
+    // aa guns rolls 1,3,2 first one hits, remaining bombers roll 1 dice each
+    bridge.setRandomSource(new ScriptedRandomSource(new int[] {1, 3, 2, 5, 4}));
+    // if we try to move aa, then the game will ask us if we want to move
+    // fail if we are called
+    final InvocationHandler handler = new InvocationHandler() {
```

Comment:
Please remove this, there is no failure here, so it is not called. We already test this in other tests. KISS, this test can be made simpler without losing test coverage.